keel-test/src/main/kotlin/com/netflix/spinnaker/keel/test/Resources.kt

```diff
@@ -122,6 +123,36 @@ fun <T : ResourceSpec> submittedResource(
     spec = spec
   )
 
+fun versionedArtifactResource(
```

Comment:
New dummy resource types that can be used in tests in `keel-core` where we need artifact info.